### PR TITLE
Simplified handling of elements with VR UN

### PR DESCRIPTION
**Pull request by KarlSjostrand**

### CODE REVIEW

File: src/main/scala/se/nimsa/dicom/data/VR.scala [hunk 1]
```diff
@@ -40,5 +40,5 @@ object VR extends Enumeration {
   final val UT = Val(0x5554, 12, ' ')
 
   private lazy final val valueOfMap: Map[Int, Value] = VR.values.map(vr => vr.code -> vr).toMap
-  def valueOf(code: Int): VR = try valueOfMap(code) catch { case _: Throwable => VR.UN }
+  def valueOf(code: Int): VR = try valueOfMap(code) catch { case _: Throwable => null }
```
Comment (KarlSjostrand) [inline src/main/scala/se/nimsa/dicom/data/VR.scala]: Make it possible to differ between when an element actually has VR UN and when VR could not be parsed

File: src/main/scala/se/nimsa/dicom/data/DicomParsing.scala [hunk 2]
```diff
@@ -172,7 +172,7 @@ trait DicomParsing {
     else if (explicitVr)
       (tag, VR.valueOf(bytesToVR(data.drop(4))))
     else
-      (tag, VR.UN)
+      (tag, Dictionary.vrOf(tag))
```
Comment (KarlSjostrand) [inline src/main/scala/se/nimsa/dicom/data/DicomParsing.scala]: The lookup for implicit VR has moved here. Was placed in `ParseFlow` before

File: src/test/scala/se/nimsa/dicom/streams/ParseFlowTest.scala [hunk 3]
```diff
@@ -447,4 +446,35 @@ class ParseFlowTest extends TestKit(ActorSystem("ParseFlowSpec")) with FlatSpecL
...
+
+    source.runWith(TestSink.probe[DicomPart])
+      .expectHeader(Tag.PatientName)
+      .expectValueChunk()
+      .expectHeader(Tag.CTExposureSequence, VR.UN, 24)
+      .expectValueChunk()
+      .expectDicomComplete()
+  }
+
+  it should "parse sequences with VR UN and implicit VR as a block of bytes" in {
```
Comment (fredrik-exini) [inline src/test/scala/se/nimsa/dicom/streams/ParseFlowTest.scala]: If VR is implicit how can the non private sequence have VR UN?

Comment (KarlSjostrand) [reply src/test/scala/se/nimsa/dicom/streams/ParseFlowTest.scala]: Right, badly explained. The sequence tag has explicit VR and is UN. The dataset(s) inside the sequence has implicit VR. I'll update the test string.

**Pull request merged**